docs/how-to-configure-new-repository.md

```diff
@@ -41,19 +41,24 @@ for the repository.
 
 ## Policies
 
-Typically `main` branch is set as default. And protected with the following
-policies:
+Typically `main` branch is set as default.
+
+The default branch MUST be protected with the following policies:
 
 1. Enable `Require pull request reviews before merging` setting. Make sure
    `Require review from Code Owners` is checked.
 2. The default setting for `Required approving reviews` is `1` approval. Individual 
-   repositores may opt-in for bigger number of required reviews.
+   repositories may opt-in for bigger number of required reviews.
 3. Every repo MUST enforce the CNCF `EasyCLA` check under the
    `Require status checks to pass before merging` section. (if the check isn't
    showing, have a CLA manager [enable the check for the
    repo](https://project.lfcla.com/#/project/a0941000002wBz4AAE/cla)). Check
    with the OpenTelemetry's [EasyCLA admins](../assets.md#user-content-opentelemetry-managed-assets:~:text=Easy%20CLA) 
-4. It is a good practice to check the `Include administrators` setting.
+4. Enable "Do not allow bypassing the above settings" so that administrators are subject
```

Comment:
Unless there's an audit log of some sort, I don't think this accomplishes much other than requiring an admin to spend another minute enabling/disabling things. Is the thinking that this prevents unintentional damage by an admin merging something without approvals by accident?

Instead, I'd prefer just making it a requirement that all admins must communicate to the other maintainers when they are doing something like this. I don't know how to enforce it, but that's just part of the trust we place in people with admin permissions.

Reply:
There is an audit log that shows who and when changed the repo settings.

I think the increased friction is desirable in this case to both prevent accidents and also to remind the person that they doing something exceptional.

I don't think there is a need to strictly enforce this. The only way to do that would be to disallow Admin permission to maintainers and I think that goes too far.

Reply:
Makes sense!

Reply:
TIL today on a non-otel repo:

Without "Do not allow bypassing the above settings", admins can git push directly to main (oops 😅).